app/src/main/assets/init.js

```diff
@@ -173,6 +173,21 @@ try {
 
 window.addEventListener('file-received', e => {
     SnapdropAndroid.saveDownloadFileName(e.detail.name, e.detail.size);
+    // vibrates after receiving each file (SnapDrop and PairDrop)
+    //SnapdropAndroid.vibrate();
+}, false);
+
+window.addEventListener('files-received', e => {
+    // vibrates after receiving all files (supported only on PairDrop)
+    SnapdropAndroid.vibrate();
+}, false);
+
+window.addEventListener('notify-user', e => {
+    if (e.detail == 'File transfer completed.') {
+        // vibrates after sending all files (PairDrop)
+        // vibrates after sending each file (SnapDrop)
+        SnapdropAndroid.vibrate();
```

Comment:
Works as intended on PairDrop, but is annoying on SnapDrop.

Reply:
Seeing that the notification text could change over time, this doesn't seem to be a very robust implementation...

Reply:
Update: PairDrop now fires a ` 'files-sent' ` event